### PR TITLE
Logo VBL

### PR DESCRIPTION
**Pull request by tenminutemail**

### CODE REVIEW

File: res/drawable/network_vbl_logo.xml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="202.8775dp"
+    android:height="53.224998dp"
+    android:viewportWidth="202.8775"
+    android:viewportHeight="53.224998">
```
Comment (grote) [inline res/drawable/network_vbl_logo.xml]: Currently, all graphics use 64dp x 64dp. Could you maybe change for source SVG to 64px x 64px (can have transparent background parts for unused height) and recreate this XML? Then it should be 64dp automatically.

File: res/drawable/network_vbl_logo.xml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+    <group
+            android:translateX="-87.132679"
+            android:translateY="-234.32115">
+        <group>
+            <group>
+                <group>
+                    <group
```
Comment (grote) [inline res/drawable/network_vbl_logo.xml]: This is quite some nested groups. Some XML creators allow you to flatten those. Could you please try?

**Comment by tenminutemail** [comment]
@grote So I added the SVG and updated the XML, but unfortunately there a still a lot of groups

**Comment by grote** [comment]
Merged, thanks!

**Comment by grote** [comment]
Btw. the group (for the next time) you can remove by clicking "ungroup objects" in inkscape.

**Pull request closed without merging**